resources/lang/en/events/relationship_events/breakup_mates.json

```
@@ -0,0 +1,38 @@
+		"m_c realizes that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} have the mentality to keep a romantic relationship right now. After a little talk with r_c, they have decided that they should just be friends, for now at least.",
+		"r_c is an amazing cat, perfect in every way, shape, and form. That's why m_c feels like {PRONOUN/m_c/subject} {VERB/m_c/aren't/isn't} a good enough mate for {PRONOUN/r_c/object} anymore. After a talk with r_c, they've decided to stay friends, giving m_c the incentive to work on {PRONOUN/m_c/self} more.",
+		"m_c isn't so sure about continuing a romantic relationship with r_c anymore. After a little talk, they have decided that they should just be friends.",
+		"m_c and r_c announced their break up today. But before their friends could offer sympathy, m_c and r_c reassure them that they left off on good terms and will still be good friends!"
+	],
+	"lost_feelings":[
+		"m_c and r_c lost feelings for each other overtime - they broke up this moon.",
+		"m_c and r_c realized that they are a lot more different from each other than they initially thought, which kinda steered them away from the relationship they once had.",
+            	"One day, m_c and r_c were crazy in love with each other. The next day, they're distant, refusing to talk to one another. The distance continues for a while, making it clear that they're no longer mates.",
+            	"Things sadly just didn't work out the way m_c and r_c thought it would... with heaviness in their hearts, they have decided to break up.",
+            	"Sometimes, good things must come to an end, m_c and r_c realize as they announce to the Clan that they’re no longer mates.",
+		"m_c isn't so sure about continuing a romantic relationship with r_c anymore. After talking with {PRONOUN/r_c/object}, they decided to break up.",
+		"m_c realizes that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} have the mentality to keep a romantic relationship right now. After a little talk with r_c, they have decided to break up."
+	],
+	"bad_breakup":[
+		"Out of the blue, m_c bluntly tells r_c that {PRONOUN/m_c/subject} {VERB/m_c/aren't/isn't} interested in {PRONOUN/r_c/object} anymore, so {PRONOUN/m_c/subject} calls off their relationship and r_c walks away heartbroken and confused.",
+            	"Some relationships turn out good, some don't, but for m_c and r_c's relationship, things really didn't turn out so well. They've broken up.", 
+            	"Things have become tense and sour in m_c and r_c’s relationship. They decided to break up before any fight breaks out between them.",
+		"r_c broke up with m_c and m_c hasn’t been quite the same since r_c left {PRONOUN/m_c/object}, refusing participate in clan duties with {PRONOUN/r_c/object} or eat while {PRONOUN/r_c/subject}{VERB/r_c/‘re/‘s} eating. m_c's trying really hard to get over {PRONOUN/r_c/object}, but can't. Break ups can be very hard sometimes…"
```

```suggestion
		"r_c broke up with m_c and m_c has taken it very hard.  {PRONOUN/m_c/subject/CAP} seem to be avoiding r_c as much as possible."
```
This one just felt a bit too long and a little convoluted.